<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.295, y1:0.727318, x2:0.636, y2:0.357955, stop:0.176136 rgba(141, 0, 0, 255), stop:0.755682 rgba(52, 69, 34, 255))
</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>571</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QGraphicsView" name="graphicsView">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="game_name_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 0), stop:0.215909 rgba(255, 188, 188, 0), stop:0.425 rgba(255, 132, 132, 0), stop:0.44 rgba(252, 128, 128, 0), stop:0.727273 rgba(255, 162, 162, 0), stop:1 rgba(255, 255, 255, 0));
color : rgb(255, 120, 133);</string>
    </property>
    <property name="text">
     <string>The Path</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(212, 0, 0, 255), stop:1 rgba(139, 0, 0, 255));
color: rgb(238, 238, 236);
font: italic 11pt &quot;Courier New&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="moves_text_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>630</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 0), stop:0.215909 rgba(255, 188, 188, 0), stop:0.425 rgba(255, 132, 132, 0), stop:0.44 rgba(252, 128, 128, 0), stop:0.727273 rgba(255, 162, 162, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(238, 238, 236);
font: italic 11pt &quot;Courier New&quot;;
</string>
    </property>
    <property name="text">
     <string>Moves: </string>
    </property>
   </widget>
   <widget class="QLabel" name="moves_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>630</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 0), stop:0.215909 rgba(255, 188, 188, 0), stop:0.425 rgba(255, 132, 132, 0), stop:0.44 rgba(252, 128, 128, 0), stop:0.727273 rgba(255, 162, 162, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(238, 238, 236);
font: italic 11pt &quot;Courier New&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSec">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.930931, y1:0.113, x2:0.950376, y2:0.091, stop:0 rgba(149, 56, 48, 255), stop:1 rgba(109, 37, 37, 255));</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.930931, y1:0.113, x2:0.950376, y2:0.091, stop:0 rgba(149, 56, 48, 255), stop:1 rgba(109, 37, 37, 255));</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserRules">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(212, 0, 0, 255), stop:1 rgba(139, 0, 0, 255));
color: rgb(238, 238, 236);
font: italic 11pt &quot;Georgia&quot;;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_success_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>630</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 0), stop:0.215909 rgba(255, 188, 188, 0), stop:0.425 rgba(255, 132, 132, 0), stop:0.44 rgba(252, 128, 128, 0), stop:0.727273 rgba(255, 162, 162, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(238, 238, 236);
font: italic 11pt &quot;Courier New&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
